extras/hook-scripts/S56glusterd-geo-rep-create-post.sh

```diff
@@ -1,8 +1,10 @@
-#!/bin/bash
+#!/ bin / bash
```

Comment:
@itisravi this is the reason the geo-rep test case was timing out with your patch. Please fix and re-run.

Reply:
@kotreshhr Thanks!